lib/Doctrine/Annotations/DocParser.php

```diff
@@ -2,7 +2,14 @@
 
 namespace Doctrine\Annotations;
 
+use function array_key_exists;
```

Comment:
Interesting, but why?

Reply:
What exactly why? Why `use function` or why `array_key_exists()`? This function will be optimized into specialized opcode in PHP 7.4, but we use `use function` for all functions either way.

Reply:
Maybe `preg_match`, `strpos` we should import too?

And one more question: is the difference between `use function array_key_exists` and `\array_key_exists`

Reply:
No, those functions are not specialized, only some specific functions are (basically [this](https://github.com/slevomat/coding-standard/blob/5.0.0/SlevomatCodingStandard/Sniffs/PHP/OptimizedFunctionsWithoutUnpackingSniff.php#L28-L61) list).

No, there is no difference, use whichever variant you prefer (although `use function` helps in terms of readability a lot).

Reply:
> No, those functions are not specialized,

Oh, sorry, I was a bit unclear: They don't need to be imported in order to be optimized by PHP because no such optimization exists. They should however be imported according to Doctrine Coding Standard. But since it's not in effect in this repo at this moment, and that this file is going to be dropped soon anyway, it's not imported.